client/client.go

```diff
@@ -550,18 +642,34 @@ type (
 		// Allows you to update the worker-build-id based version sets for a particular task queue. This is used in
 		// conjunction with workers who specify their build id and thus opt into the feature.
 		// WARNING: Worker versioning is currently experimental
```

Comment:
nit: I think we should remove the experimental warning since I don't think something should be both experimental and deprecated.

Reply:
I agree that in general we should not deprecate experimental features and just remove them instead, but in this case we decided to keep an experimental feature in deprecated mode for one release...

Reply:
I am not asking to remove the API, just remove the experimental tag on it.

Reply:
I understand, I think that API is both experimental and deprecated, so having both tags seems more accurate. Is there a tooling issue of having both tags?

Reply:
It can't be both experimental and deprecated IMO, but I don't have a strong opinion on leaving both. It's just confusing.